src/components/ListLayout.jsx

```diff
@@ -61,6 +61,54 @@ const ListLayout = ({ items, localToken }) => {
     return timeCheck;
   }
 
+  // updates isActive property of item to true if item has 2+ purchases and has been purchased within calculated estimate
```

Comment:
also amazing comment, explains what's not clear and doesn't repeat what's obvious in the code